L1Trigger/L1TCalorimeter/plugins/L1TCaloUpgradeToGCTConverter.cc

```diff
@@ -21,7 +21,9 @@
 
 using namespace l1t;
 
-L1TCaloUpgradeToGCTConverter::L1TCaloUpgradeToGCTConverter(const ParameterSet& iConfig)
+L1TCaloUpgradeToGCTConverter::L1TCaloUpgradeToGCTConverter(const ParameterSet& iConfig):
+    bxMin_(iConfig.getParameter<int>("bxMin")),
+    bxMax_(iConfig.getParameter<int>("bxMax"))
```

Comment:
It would be usefull to code a PROPER fillDescriptions(.) method for this class,
which would take care of adding the new parameters to older configs not
containing them.